grape-entity.gemspec

```diff
@@ -18,8 +18,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = 'grape-entity'
 
-  s.add_runtime_dependency 'activesupport', '>=4.0'
-  # FIXME: remove dependecy
+  s.add_runtime_dependency 'activesupport', '>= 3.0.0'
```

Comment:
please can you change the lines, `FIXME` should be above `multi_json`, thanks

Reply:
Will do.